tests/unittests/OperatorTest.cpp

```diff
@@ -3113,21 +3113,21 @@ static void testFlip(glow::PlaceholderBindings &bindings, glow::Module &mod,
 
 /// Test Flip 1D with Int8.
 TEST_P(OperatorTest, Flip1D_Int8) {
-  ENABLED_BACKENDS(Interpreter, CPU);
+  ENABLED_BACKENDS("Interpreter", "CPU");
```

Comment:
I thought we deprecated `ENABLED_BACKENDS` in favor of `CHECK_IF_ENABLED`?  (Am I wrong?)

Reply:
I was not following this discussion closely. This PR only mechanically modified all existing places to use string literals instead of custom symbolic names introduced in `BackendTestUtils.h`. If we want to migrate `ENABLED_BACKENDS` to `CHECK_IF_ENABLED` we could do it in a separate PR for all the uses of `ENABLED_BACKENDS`.